vc_zoom/indico_vc_zoom/client/index.js

```diff
@@ -0,0 +1,29 @@
+// This file is part of the Indico plugins.
+// Copyright (C) 2020 CERN and ENEA
+//
+// The Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License;
+// see the LICENSE file for more details.
+
+import {handleAxiosError, indicoAxios} from 'indico/utils/axios';
+
+document.addEventListener('DOMContentLoaded', async () => {
+  $('.vc-toolbar').dropdown({
+    positioning: {
+      level1: {my: 'right top', at: 'right bottom', offset: '0px 0px'},
+    },
+  });
+
+  document.querySelectorAll('.vc-toolbar .action-make-host').forEach(elem => {
+    elem.addEventListener('click', async () => {
+      let killProgress = IndicoUI.Dialogs.Util.progress();
+      try {
+        await indicoAxios.post(elem.dataset.href);
+        window.location.reload();
```

Comment:
Does anything change on the page when making yourself host (besides maybe hiding this action)? If not we could just hide/remove it without reloading the page.

Reply:
The details of the booking should also be updated (there's a host field in them).